pkg/flag/db_flags.go

```diff
@@ -16,7 +16,9 @@ var (
 		Name:       "reset",
 		ConfigName: "reset",
 		Usage:      "remove all caches and database",
+		Removed:    `Use "trivy clean --all" instead.`,
 	}
+	// Deprecated
```

Comment:
Looks like it should be before `reset` flag.

Reply:
Thanks. I was confused. Fixed in https://github.com/aquasecurity/trivy/pull/6993/commits/01f901c93fc30009d0b57ac28908b6aa0442885e